### ISSUE TITLE
Hyper fails to install any plugins on Arch

### ISSUE THREAD

**Issue by BlackHoleFox** [zeit/hyper#3112]
- [X] I am on the [latest](https://github.com/zeit/hyper/releases/latest) Hyper.app version
- [X] I have searched the [issues](https://github.com/zeit/hyper/issues) of this repo and believe that this is not a duplicate

- **OS version and name**: Arch Linux 2018.07.01
- **Hyper.app version**: 2.0.0 Stable
- **The issue is reproducible in vanilla Hyper.app**: Is Vanilla

## Issue
Whenever I try to install any plugin in Hyper, for example `hyper i verminal`, it errors out and opens another terminal for some reason, giving this error message (Using the AppImage):
```console
[shadus@machine~]$ hyper i verminal
running in prod mode
electron will open file:///tmp/.mount_hyper-z5B0ZT/app/resources/app.asar/index.html
Setting Hyper as default client for ssh:// protocol
Failed to symlink Hyper CLI { Error: EACCES: permission denied, unlink '/usr/local/bin/hyper'
  errno: -13,
  code: 'EACCES',
  syscall: 'unlink',
  path: '/usr/local/bin/hyper' }Checking for notification messages
No matching notification messages
```
I saw a closed issue about this but it seems no solution was ever found for Linux users?

Thanks!

**Comment by JoshuaScript** [comment]
Looks like you're having a permissions error, so a temporary fix may be to run the command as `sudo`.

**Comment by thedonquixotic** [comment]
@BlackHoleFox Hey did you ever get a fix to this?

**Comment by BlackHoleFox** [comment]
I have not. Running it as sudo does no good. Currently just using the default KConsole still :/

**Comment by thedonquixotic** [comment]
Oooh, that's no good. I'm running Windows for my work computer and rely on Hyper for it. And I seem to have completely borked things up even worse trying to get Oh-my-zsh installed and stuff. yeesh. 

Also kconsole so you're using KDE? I just started using Kubuntu! It's my first serious go at using Linux for home desktop stuff. Anyway, that's off topic, but KDE is really nice. Wish I could just go full time with it uuuugh.

**Comment by BlackHoleFox** [comment]
Yeah its slightly frustrating since they provide no other dist method for Linux officially then the AppImage

**Comment by thedonquixotic** [comment]
@BlackHoleFox So stripped out everything and tried to do a totally clean install of WSL bash, and zsh to see if I could figure out what was causing this problem with Hyper. 

I think it's because I've symlinked the hyper files. 

I use symlinks so I can collect all my config files together in a dropbox folder. I don't know exactly how AppImage works but I do believe it uses symlinks in some way to help fake the different distro environments. 

I also believe I remember having some issues with permissions over symlinks when I was installing bash etc. though I don't know why that would be an issue since I'm installing hyper on my Windows machine, and only running bash from inside it so Hyper should have normal permissions to all my config files and symlinks etc. 

Also I tried using hardlinks and those are basically seen as identical to the system as the file itself, from my understanding so... not sure 

@JoshuaScript  are you on the Zeit team? You got any thoughts on this?

**Comment by Yash-Garg** [comment]
Same issue here too

**Comment by hcallaway** [comment]
I'm having the same issue as well, but with a slightly different output. Occurs any time I try to install a plugin, and opens a new terminal.

Command: `hyper i hyper-active-tab`
Output:
`Disabling CHromium GPU blacklist
running in prod mode
electron will open file://tmp/.mount_hyper_GLto67/app/resources/app.asar/index.html
Setting Hyper as default client for ssh:// protocol
Checking for notification messages
No matching notification messages
`
The output does not stop as a process, but opens a new terminal that starts at the home directory. Once that terminal is closed the process stops, plugin not install, and starts a new line.

**Comment by aurerua** [comment]
Similar problem to @hcallaway  although the output is slightly different:
```
Disabling Chromium GPU blacklist
running in prod mode
electron will open file:///usr/lib/hyper/resources/app.asar/index.html
Checking for notification messages
No matching notification messages
```
I haven't found a solution yet, but adding the plugin manually to the `plugins` array in `.hyper.js` does work.

**Comment by bet4it** [comment]
Try `yay -S hyper-git` on Arch Linux

**Comment by The-God-coder** [comment]
I have the same problem is there a fix yet?
Garuda Linux - based off Arch using the AUR package

**Comment by BitBiro** [comment]
Same issue here - i have Archlinux.
